README.rst

```diff
@@ -58,12 +58,178 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+In order to use the **nidaqmx** package, you must have at least one DAQ (`Data Acquisition <https://www.ni.com/en/shop/data-acquisition.html>`_)
+device installed on your system. Both physical and simulated devices are supported. The examples below use an X Series DAQ device
+(e.g.: PXIe-6363, PCIe-6363, or USB-6363).
+You can use **NI MAX** or **NI Hardware Configuration Utility** to verify and configure your devices.
+
+Finding and configuring device name in **NI MAX**:
+
+.. image:: https://raw.githubusercontent.com/ni/nidaqmx-python/ca9b8554e351a45172a3490a4716a52d8af6e95e/max_device_name.png
```

Comment:
FYI, this is using a commit hash to refer to the file version. If we update the images later, we must make sure to update the link to use the new commit hash.

The alternative is to use a branch name or tag, but this has different problems. Using "master" would always link to the newest version of the file, even if you're looking at an old version of the README. Also, moving or renaming the file would break the link. Using a tag introduces a chicken-and-egg problem, where the link is broken until we create a release with that tag.

Given these tradeoffs, I guess the commit hash is the easiest solution to manage.

Cc: @zhindes